Makefile

```diff
@@ -348,8 +348,8 @@ endif
 
 .PHONY: install
 
+# Run the unit-test binary and run individual sample example program binaries.
 run-tests: run-unit-tests run-test-code
-	./test.sh
```

Comment:
Delete this, otherwise, `test.sh` will invoke the newly added `makefile_test.py`, leading to endless recursion in these testing cycles.